javascript/ql/test/library-tests/ModelGeneration/ModelGeneration.expected

```diff
@@ -1,4 +1,11 @@
 typeModel
+| (aliases).Alias1 | aliases | Member[Alias1] |
```

Comment:
In this case `(aliases).Alias1` is chosen as the name, but a human would probably have picked AliasedClass instead. Can you highlight why this happened?

Reply:
The `EndpointNaming` library tries to choose one of the names as the "primary" name. When simply viewed as an graph, there is no difference between the names other than, well, the name. We could add a heuristic around names that match their local declaration site, but that's not there right now.

Reply:
Given that it's mostly a readability issue when we (developers) need to debug model generation, I think we can classify this as a nitpick. I think we might enjoy having good names ourselves, but we can add it whenever we feel like it :+1: